src/main/scala/org/renci/robocord/json/CORDJsonReader.scala

```diff
@@ -67,31 +67,31 @@ case class ArticleRefEntry(
 )
 
 case class ArticleBibEntry(
-  ref_id: String,
+  ref_id: Option[String],
   title: String,
   authors: Seq[Author],
   year: Option[Int],
   venue: String,
   volume: String,
   issn: String,
-  pages: String,
+  pages: Option[String],
   other_ids: Map[String, Seq[String]]
 )
 
 case class Article(
   paper_id: String,
   metadata: ArticleMetadata,
-  `abstract`: Seq[ArticleAbstract],
+  `abstract`: Option[Seq[ArticleAbstract]],
```

Comment:
I think I may be missing the full context, but `Option[Seq[T]]` seems unusual. Typically you could just have an empty `Seq` if not present.

Reply:
This is to do with how Circe parses JSON documents: `Seq[...]` causes parsing to fail if `abstract` is absent, but `Option[Seq[...]]` tells it that `abstract` is optional, and so parsing can proceed even if it `abstract` fails. Let me know if you think I should document this better, but I think it's mostly readable except for the `Option[Seq[...]]` situations.

Reply:
Oh, that makes sense! I wasn't thinking about parsing. I have a bunch of the same here: https://github.com/INCATools/dosdp-tools/blob/master/src/main/scala/org/monarchinitiative/dosdp/DOSDP.scala

Would be useful to one day dig into see if there is a concise way to have Circe give an empty list if the key is missing.